cypress/integration/update_spec.js

```diff
@@ -37,16 +37,16 @@ describe('e2e test typing transaction ID and choosing "yes" to claim gift aid on
         cy.get('#field-input--transactionId').clear()
         cy.get('#field-input--firstname').click()
         cy.get('#field-error--transactionId > span').should('contain','Please fill in your transaction id')
-        cy.get('#field-input--transactionId').type('#£$^')
-        cy.get('#field-error--transactionId > span').should('contain','This does not match a transaction ID in our system, please check your donation confirmation email or letter')
         cy.get('#field-input--transactionId').clear().type('d-BEXd501')
-        cy.get('#field-error--transactionId > span').should('contain','This does not match a transaction ID in our system, please check your donation confirmation email or letter')
+        cy.get('#field-error--transactionId > span').should('be.not.visible')
```

Comment:
I know the tests passed on the PR, but is this syntax really correct?
Thought it was `not.be.visible` or is cypress flexible like that?

Reply:
I didn't really change anything here. `be.not.visible` is used in multiple lines, simply just duplicated the test.